src/silverlabnwb/metadata.yaml

```diff
@@ -12,66 +12,63 @@ people:
     name: Hana Ros
     scopus_id: 33368223900
 
-# Settings for particular experiments come next. This allows easy reuse of methods
-# information, etc. that is common to many sessions. Each experiment is identified
-# by a short label that is referenced in the session section at the end of this file.
```

Comment:
Does moving this after the next line (between `experiment:` and `template:`) cause problems in the end?

Reply:
Nope, simply forgot to put them there.